Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -2372,6 +2372,8 @@ class WebPageProxy final : public API::ObjectImpl<API::Object::Type::Page>, publ
 
     void addConsoleMessage(WebCore::FrameIdentifier, JSC::MessageSource, JSC::MessageLevel, const String&, std::optional<WebCore::ResourceLoaderIdentifier> = std::nullopt);
 
+    std::optional<WebsitePoliciesData> mainFrameWebsitePoliciesData() const { return m_mainFrameWebsitePoliciesData; }
```

Comment:
const std::optional<WebsitePoliciesData>&
Avoid unnecessary copies.
Also see below about making this a unique_ptr.